src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from 'classnames';
+import { useEffect } from 'react';
+import { ErrorMessage } from '../../types/Enums';
+
+type Props = {
+  errorMessage: ErrorMessage | null,
+  isError: boolean,
+  setIsError: (value: boolean) => void,
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  isError,
+  setIsError,
+}) => {
+  useEffect(() => {
+    const hideMessage = setTimeout(() => {
+      setIsError(false);
+    }, 3000);
+
+    return () => {
+      clearInterval(hideMessage);
+    };
+  }, [isError]);
```

Comment:
you can leave the dependencies array empty.

Reply:
Great job! 🚀

Reply:
after the second call of this error, it does not disappear 🤔